tests/functional/test_accounts.py

```diff
@@ -364,6 +364,14 @@ def test_account_comparison_to_non_account(core_account):
     assert core_account != "foo"
 
 
+def test_add_new_account(test_accounts):
+    assert len(test_accounts) == DEFAULT_NUMBER_OF_TEST_ACCOUNTS
+
+    test_accounts.containers["test"].add_account()
+
+    assert len(test_accounts) == DEFAULT_NUMBER_OF_TEST_ACCOUNTS + 1
+
```

Comment:
Would be nice to also check that the returned account is the last one in the container.

Reply:
added